### ISSUE TITLE
Update Adoptium #build daily Slack status with Reproducible build results

### ISSUE THREAD

**Issue by andrew-m-leonard** [adoptium/ci-jenkins-pipelines#1091]
TRSS now has a "Rebuild_Same_JDK_Reproducibility_Test_0", which reports a "Reproducibility %" : 
https://trss.adoptium.net/deepHistory?testId=66b48b8dd24e1b006eaa48d5

We should add a new line to the #build daily Slack status reporting the latest EA reproducibility for the reproducible versions and platforms. So currently:
- jdk21u : x64Linux, aarch64Linux, ppc64leLinux, x64Windows, x64Mac, aarch64Mac

Status script: https://github.com/adoptium/ci-jenkins-pipelines/blob/master/tools/nightly_build_and_test_stats.groovy

```
Comparing /home/user/jdk-21.0.4+7 with /cygdrive/c/workspace/openjdk-build/workspace/build/openjdkbuild/images/jdk ... output to file: reprotest.diff
Number of differences: 0
ReproduciblePercent = 100 %
```

**Comment by adamfarley** [comment]
I've been working on this task in the repository branch [here](https://github.com/adamfarley/ci-jenkins-pipelines/compare/master...adamfarley:ci-jenkins-pipelines:add_reproducibility_percentage_to_daily_build_summary).

I've set up a temporary jenkins job [here](https://ci.adoptium.net/job/adam_temp_job/) to test it as I go along.

The results are sent to my Slack Member ID as a direct message, to avoid swamping the build channel.

Example of last successful output:
```
![image](https://github.com/user-attachments/assets/a7b54e0d-0a42-4c1a-a374-fb76092fa35a)
```

This was using dummy percentages, for the sake of testing the formatting.

I'm currently fixing bugs in the percentage parsing code.

**Comment by adamfarley** [comment]
Update: The code works! We now gather the correct percentages from the correct locations.

Intentionally untested comparable builds that will be tested in the future are labelled "NA".

The output is as depicted below. I'm working with Andrew Leonard to refine the output to make it smaller and most useful.

Pending tasks:
- [ ] To remove the NA builds from the Slack output.
- [ ] To replace NA in the breakdown (part of the groovy jenkins job output) with something more useful.
- [ ] To replace "good" and "bad" with a list of platforms + percentages (? % if no percentage available).
- [ ] To remove the hard-coded specific-pipeline TRSS URL, and re-test.
  - I did this to make sure I'm testing the new code with a pipeline that ran tests.

Also, I've added the ability for us to specify different test names + locations by platform.

https://github.com/adoptium/temurin-build/blob/master/test/system/reproducibleCompare/playlist.xml

![image](https://github.com/user-attachments/assets/a1b9066f-619a-49be-bfda-00c30bbcdc29)
